src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArr = sourceString
```

Comment:
The variable name `sourceArr` is not very descriptive. Consider using a more meaningful name to improve code readability, as per the checklist item: 'Avoid variable namings like `array`, `object`, `arr`, `obj`, `result`.'